com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/StepsUtil.java

```diff
@@ -76,9 +76,9 @@
 
 public class StepsUtil {
 
-    private static org.apache.log4j.Logger getAppLogger(DelegateExecution context, String appName,
+    protected static org.apache.log4j.Logger getCustomLogger(DelegateExecution context, String customName,

```

Comment:
You can keep it more simple with:
getCustomLogger -> getLogger
customName -> name